src/lib/util/ProductJourneyIcon.spec.ts

```diff
@@ -0,0 +1,21 @@
+import fs from 'fs'
```

Comment:
Open to suggestions where to put test files which are not related to one specific file/folder 😄

Reply:
are those still unit tests then?
maybe even put them into its separate subdirectory /test/...